scripts/west_commands/runners/pyocd.py

```diff
@@ -69,15 +65,11 @@ def do_add_parser(cls, parser):
 
         parser.add_argument('--daparg',
                             help='Additional -da arguments to pyocd tool')
-        parser.add_argument('--flashtool', default='pyocd-flashtool',
```

Comment:
Why get rid of this? The user might have installed the bleeding edge version with the `pyocd` entry point in a random directory that is not on the usual sys.path.

```
parser.add_argument('--pyocd', default='pyocd', help='path to pyocd tool, default is pyocd')
```

Then we could save `args.pyocd` as an attribute and use it instead of `['pyocd']` down in `flash()` and `debug_debugserver()`. The test cases which pass values for all parameters, even the ones with defaults, could then set `--pyocd test-pyocd` and we could test that `test-pyocd` is used instead of `pyocd` there.

Reply:
Done